WordPress/Classes/Models/Theme.m

```diff
@@ -30,6 +39,35 @@ + (NSString *)entityName
     return NSStringFromClass([self class]);
 }
 
+#pragma mark - Links
+
+- (NSString *)customizeUrl
+{
+    return [self.blog adminUrlWithPath:ThemeAdminUrlCustomize];
+}
+
+- (NSString *)detailsUrl
+{
+    NSString *homeUrl = [[self.blog homeURL] stringByReplacingOccurrencesOfString:HomeUrlScheme withString:@""];
```

Comment:
Maybe [this helper](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Categories/NSString+Helpers.h#L27) could save you few lines?